packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/NewSandboxButton.tsx

```diff
@@ -1,14 +1,20 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import PlusIcon from 'react-icons/lib/go/plus';
-import { inject, hooksObserver } from 'app/componentConnectors';
+
+import { useOvermind } from 'app/overmind';
+
 import { Action } from './Action';
 
-export const NewSandboxButton = inject('signals')(
-  hooksObserver(({ signals: { modalOpened } }) => (
+export const NewSandboxButton: FunctionComponent = () => {
+  const {
+    actions: { modalOpened },
+  } = useOvermind();
+
+  return (
     <Action
-      onClick={() => modalOpened({ modal: 'newSandbox' })}
-      tooltip="New Sandbox"
       Icon={PlusIcon}
+      onClick={() => modalOpened({ message: null, modal: 'newSandbox' })}
```

Comment:
@christianalfoni Just like https://github.com/codesandbox/codesandbox-client/pull/2544#discussion_r328837587

Maybe make `message` optional? 🤔

Reply:
message is now optional :)